tests/wallet/test_wallet.c

```diff
@@ -22,6 +22,15 @@
 #define FAUCET_PORT 8091
 #define FAUCET_IS_HTTPS 0
 
+// predefined coin types
+#if defined(NETWORK_TYPE_SHIMMER)
+#define SLIP44_COIN_TYPE SLIP44_COIN_TYPE_SHIMMER
+#elif defined(NETWORK_TYPE_MAINNET)
+#define SLIP44_COIN_TYPE SLIP44_COIN_TYPE_IOTA
+#else
+#define SLIP44_COIN_TYPE SLIP44_COIN_TYPE_TEST
+#endif
+
```

Comment:
Are we going to use `NETWORK_TYPE_SHIMMER` and  `NETWORK_TYPE_MAINNET` also in some other places in the code in the future?

Reply:
nope, a wallet application can define an arbitrary coin type. predefined types here are for this unit test which can cooperate with default networks manually, as needed. actually, the coin type won't affect the test result.